WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -128,6 +132,28 @@ - (XCElementSnapshot *)fb_parentCellSnapshot
     }
     return targetCellSnapshot;
 }
+
+- (CGRect) fb_visibleFrameWithFallback
+{
+  CGRect thisVisibleFrame = [self visibleFrame];
+  if (CGRectIsEmpty(thisVisibleFrame))
+  {
```

Comment:
Please move opening brackets to the same line where `if` starts to align with the common source style